### DISCUSSION TITLE
Working environments

### DISCUSSION THREAD

**Discussion by bquast** [openai/whisper#28] [General]
Just to post environments in which Whisper works without issues.

Mine:

* Arch Linux 5.19.10-arch1-1
* Python 3.10.7
* PyTorch 1.12.1-4

**Comment by Prox501** [comment]
It doesn't work for me.

INTEL MKL ERROR: /opt/intel/mkl/lib/intel64/libmkl_vml_def.so.2: undefined symbol: mkl_lapack_dspevd.
Intel MKL FATAL ERROR: cannot load libmkl_vml_def.so.2.

The library is there though, just doesn't have that symbol for some reasons.

Arch Linux 5.19.8-hardened2-1-hardened
Python 3.10.7
PyTorch 1.12.1-4

Could you list any packages relevant to libmkl_vml_def.so library?

**Reply by Prox501** [reply]
Tried both 'intel-oneapi-basekit' and 'intel-oneapi-mkl' packages, neither provides this symbol. Considering that it somehow works for you have me baffled since we both are likely downloading/installing the exact same thing here.

**Comment by FurkanGozukara** [comment]
I made it work on windows 10

installed python 3.9.9

then just run this code and everything works fine  : pip install git+https://github.com/openai/whisper.git
